### ISSUE TITLE
Support additionally composer config setting keys

### ISSUE THREAD

**Issue by pschirch** [composer/composer#2384]
Hi,

we actually automate our devops processes. In this case we have too manipulate `composer.json` settings. Unfortunately

```bash
$ composer config [setting-key] [setting-value]
```

does not support the following needed setting-keys:

* prefer-stable
* minimum-stability
* extra.symfony-assets-install

```bash
[InvalidArgumentException]
Setting prefer-stable does not exist or is not supported by this command
```

It would be a pleasure, if the composer config command accepts all possible setting keys.

Any chance to implement this?

**Comment by spengilley** [comment]
Is this still required? Very old issue so might be worth closing if not...

**Comment by curry684** [comment]
I think the current stance should be "every scripting language in the world supports easy JSON modification so let's not complicate the Composer codebase with it".

**Comment by alcohol** [comment]
I still think it would be desirable to add support for those keys through the `config` command.

**Comment by Seldaek** [comment]
Yeah but there is already #2415 implementing some of it.. probably needs to be added to the config command somehow instead of a new command though. It'll need some work as it's so old. After 1.0 is out it'll be time to bring all these old PRs back to life or close them for good :)

**Comment by alcohol** [comment]
So before 1.0 you'd rather leave this open? Or is this something that would be welcome in 1.0 also?

**Comment by Seldaek** [comment]
No we're in beta we shouldn't make any big changes that aren't essential.

**Comment by Seldaek** [comment]
Fixed by 135783299af0281db918c103cceb2b202ae154f2
